L1Trigger/Phase2L1ParticleFlow/src/puppi/linpuppi_ref.cpp

```diff
@@ -13,8 +13,11 @@
 #include "FWCore/ParameterSet/interface/allowedValues.h"
 #include "FWCore/Utilities/interface/transform.h"
 #include "FWCore/Utilities/interface/Exception.h"
+#include "FWCore/ParameterSet/interface/FileInPath.h"
```

Comment:
THis will need to be protected and a string used outside CMSSW....
Actually, most likely the whole NNVtx can not be compiled in the standalone HLS testbench...isn't it?

Reply:
It is currently protected within a `#ifdef CMSSW_GIT_HASH`. I am not sure what can/can not be complied on the HLS test bench: I can discuss with you offline.